attributes/default.rb

```diff
@@ -1,7 +1,7 @@
-default['pgd']['pgd_path'] = "/home/vagrant/pgd"
-default['pgd']['virtualenv_path'] = "/home/vagrant/vrt"
-default['pgd']['user'] = "vagrant"
-default['pgd']['group'] = "vagrant"
+default['pgd']['pgd_path'] = "/home/centos/pgd"
```

Comment:
Are these set in addition to or as a replacement for the attributes in default.rb?